src/workloads/execution/TimeSeriesRangeDelete.yml

```diff
@@ -0,0 +1,123 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This test establishes a baseline for time-based range deletes on time-series collections.
+
+  We insert 1000 independent series with 100 buckets in each series, and each bucket has 100
```

Comment:
That is correct Fausto. The updated comments are in the header. It seems like people didn't see it. Maybe need to pull it so that it will show up in vscode?

Reply:
I was referring to https://github.com/mongodb/genny/pull/843/files#diff-b789006c21bde7a9b14d1f5d144373ad4adcff0348dca29fb365b1d21994f6f5R9-R15. But I added more inline comments to explain how specific data generation was done.